website/static/js/pages/project-settings-page.js

```diff
@@ -147,6 +151,34 @@ $(document).ready(function() {
                 }
             }
         });
+        };
+        // unchecked warning adopted from profile-settings-addons-page.js
+        if(unchecked.length > 0) {
+            var uncheckedText = $.map(unchecked, function(el){
+                return ['<li>', $(el).closest('label').text().trim(), '</li>'].join('');
```

Comment:
Note to self: although this is constructing an HTML string with variables, the values in question are addon names that we control (rather than user inputs). No security worries here.